digdag-core/src/main/java/io/digdag/core/schedule/SchedulerManager.java

```diff
@@ -82,10 +114,44 @@ private Scheduler getScheduler(Config schedulerConfig, ZoneId workflowTimeZone)
             c.set("_command", command);
         }
 
+        for(String param : BUILT_IN_SCHEDULE_PARAMS){
+            usedKeys.add(param);
+        }
+
         SchedulerFactory factory = types.get(type);
         if (factory == null) {
             throw new ConfigException("Unknown scheduler type: " + type);
         }
+
+        if (!usedKeys.isAllUsed()) {
+            shouldBeUsedKeys.removeAll(usedKeys);
+            if (!shouldBeUsedKeys.isEmpty()) {
+                for (String key: shouldBeUsedKeys) {
+                    if (throwUnusedKeys)
+                        throw new UnusedConfigException(getWarnUnusedKey(key, usedKeys));
```

Comment:
Showing logging in this loop is good but throwing exception has a downside. Because throwing an exception will leave from this loop, users can know only one unused key at a time.
How about moving `throw` before the loop if `throwUnusedKeys` is set? We can concatenate messages of `getWarnUnusedKey` with `\n` as the message of UnusedConfigException.